chains/solana/contracts/programs/ccip-router/README.md

```diff
@@ -17,10 +17,9 @@ Collapsed Router + OnRamp + OffRamp Contracts Implementation for CCIP in SVM.
 
```

Comment:
Is this readme still valid? I remember we used to write all the TODOs here, but I don't know if that's still the case

Reply:
not sure - but i can go through and remove things since it's probably out of date